sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCV2Suite.scala

```diff
@@ -723,6 +731,53 @@ class JDBCV2Suite extends QueryTest with SharedSparkSession with ExplainSuiteHel
     checkAnswer(df5, Seq(Row(6, "jen", 12000, 1200, true)))
   }
 
+  test("scan with filter push-down with datetime functions") {
```

Comment:
We can put these. tests after `test("scan with filter push-down with ansi mode") {`.